deploy/examples/cluster-test.yaml

```
@@ -19,6 +19,7 @@ data:
     mon_warn_on_pool_no_redundancy = false
     bdev_flock_retry = 20
     bluefs_buffered_io = false
+    mon_data_avail_warn = 0G
```

0G would completely disable the warning, right? How about if we leave some buffer for warning? 
```suggestion
    mon_data_avail_warn = 500M
```